### ISSUE TITLE
Search bar

### ISSUE THREAD

**Issue by Ratne** [HyunSeob/hexo-theme-overdose#19]
wath about to implement a search system?

**Comment by HyunSeob** [comment]
Yeah, I will implement that. But it can be needed time. Are you okay if you use this theme without a search bar? When this feature be implemented complete, I notify you.

**Comment by Ratne** [comment]
yes i'm just migrating right now ;)

**Comment by yeolwoosung** [comment]
I'm looking forward to this feature, too! 
please notify to me!!  :)

**Comment by HyunSeob** [comment]
@SungYeolWoo 네 알려드릴게요 ㅎㅎ 게을러서 좀 걸릴 거 같아요 😪

**Comment by yeolwoosung** [comment]
@HyunSeob icarus의 insight search가 괜찮아보였어요! 아무쪼록 커스터마이징 계속 기대하겠습니다! 💪

**Comment by miSunLaughing** [comment]
I'm looking forward to this feature, too!
please notify to me!! :)
